frame/pablo/src/benchmarking.rs

```diff
@@ -0,0 +1,275 @@
+		let owner: T::AccountId = whitelisted_caller();
+		  let pool_id = create_stable_swap_pool::<T>(owner.clone(), CurrencyPair::new(usdc, usdt));
+		  let unit = 1_000_000_000_000;
+		  // 100_000_000 USDC , 100_000_000 USDT
+		  let initial_usdc: T::Balance = (100_000_000_u128 * unit).into();
+		  let initial_usdt: T::Balance = (100_000_000_u128 * unit).into();
+		  // Mint the tokens
+		  assert_ok!(T::Assets::mint_into(usdc, &owner, initial_usdc));
+		  assert_ok!(T::Assets::mint_into(usdt, &owner, initial_usdt));
+		  // Add the liquidity
+		  assert_ok!(<Pablo<T> as Amm>::add_liquidity(
+			  &owner,
+			  pool_id,
+			  initial_usdc,
+			  initial_usdt,
+			  0.into(),
+			  false
+		  ));
+		  let user = account("user", 0, 0);
+		  assert_ok!(T::Assets::mint_into(usdt, &user, (1000_u128 * unit).into()));
+		  // buy 1000 USDC
+	 }: _(RawOrigin::Signed(user), pool_id, usdc, (1000_u128 * unit).into(), false)
+
+	sell {
+		let usdc: T::AssetId = 100.into();
+		let usdt: T::AssetId = 101.into();
+		let owner: T::AccountId = whitelisted_caller();
+		  let pool_id = create_stable_swap_pool::<T>(owner.clone(), CurrencyPair::new(usdc, usdt));
+		  let unit = 1_000_000_000_000;
+		  // 100_000_000 USDC , 100_000_000 USDT
+		  let initial_usdc: T::Balance = (100_000_000_u128 * unit).into();
+		  let initial_usdt: T::Balance = (100_000_000_u128 * unit).into();
+		  // Mint the tokens
+		  assert_ok!(T::Assets::mint_into(usdc, &owner, initial_usdc));
+		  assert_ok!(T::Assets::mint_into(usdt, &owner, initial_usdt));
+		  // Add the liquidity
+		  assert_ok!(<Pablo<T> as Amm>::add_liquidity(
+			  &owner,
+			  pool_id,
+			  initial_usdc,
+			  initial_usdt,
+			  0.into(),
+			  false
+		  ));
+		let user = account("user", 0, 0);
+		assert_ok!(T::Assets::mint_into(usdc, &user, (1000_u128 * unit).into()));
+		// sell 1000 USDC
+	 }: _(RawOrigin::Signed(user), pool_id, usdc, (1000_u128 * unit).into(), false)
+
+	 swap {
+		let usdc: T::AssetId = 100.into();
+		let usdt: T::AssetId = 101.into();
+		let owner: T::AccountId = whitelisted_caller();
+		  let pool_id = create_stable_swap_pool::<T>(owner.clone(), CurrencyPair::new(usdc, usdt));
+
+		  let unit = 1_000_000_000_000;
+		  // 100_000_000 USDC , 100_000_000 USDT
+		  let initial_usdc: T::Balance = (100_000_000_u128 * unit).into();
+		  let initial_usdt: T::Balance = (100_000_000_u128 * unit).into();
+		  // Mint the tokens
+		  assert_ok!(T::Assets::mint_into(usdc, &owner, initial_usdc));
+		  assert_ok!(T::Assets::mint_into(usdt, &owner, initial_usdt));
+		  // Add the liquidity
+		  assert_ok!(<Pablo<T> as Amm>::add_liquidity(
+			  &owner,
+			  pool_id,
+			  initial_usdc,
+			  initial_usdt,
+			  0.into(),
+			  false
+		  ));
+		  let user = account("user", 0, 0);
+		  assert_ok!(T::Assets::mint_into(usdt, &user, (1000_u128 * unit).into()));
+		  // swap 1000 USDC
+	 }: _(RawOrigin::Signed(user), pool_id, CurrencyPair::new(usdc, usdt), (1000_u128 * unit).into(), 0.into(), false)
+}
+
```

Comment:
should we add buy/sell/swap for LBP or uniswap?

Reply:
All of them are of `O(1)` complexity in terms of code, Then considering the maximum number of database operations I added stableSwap benchmarks only. For LBP I also added create/add_liquidity/remove_liquidity benchmarks as the code is some operations that are different from the other two dexes.